jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -34,7 +34,7 @@ pipeline {
             agent { label AGENT_LABEL }
             steps {
                 script {
-                    currentBuild.description = TEST_MANIFEST
+                    currentBuild.description = BUILD_MANIFEST_URL
```

Comment:
Out of curiosity, what's the purpose of changing this here? I didn't see it's used elsewhere.

Reply:
We discussed this yesterday to show build number, version, arch, etc.